at_secondary/at_persistence_secondary_server/lib/src/compaction/at_compaction_stats_impl.dart

```diff
@@ -0,0 +1,95 @@
+import 'dart:convert';
+
+import 'package:at_commons/at_commons.dart';
+import 'package:at_persistence_secondary_server/at_persistence_secondary_server.dart';
+import 'package:at_utils/at_logger.dart';
+
+class AtCompactionStatsImpl implements AtCompactionStats{
+
+  static final AtCompactionStatsImpl _singleton = AtCompactionStatsImpl._internal();
+
+  AtCompactionStatsImpl._internal();
+
+  factory AtCompactionStatsImpl.init(atSign){
+    _singleton.atSign = atSign;
+    _singleton.keyStore = SecondaryPersistenceStoreFactory.getInstance()
+        .getSecondaryPersistenceStore(atSign)
+        ?.getSecondaryKeyStore();
+    return _singleton;
+  }
+
+  factory AtCompactionStatsImpl.getInstance(atLogType){
+   _singleton.atLogType = atLogType;
+    return _singleton;
+  }
+
+  //AtCompactionStatsImpl.setLogType(this.atLogType);
+
+  late String atSign;
+  late AtLogType atLogType;
+  late DateTime compactionStartTime;
+  late Duration compactionDuration;
+  late int sizeBeforeCompaction;
+  late int sizeAfterCompaction;
+  late String compactionStatsKey;
+  late DateTime lastCompactionRun;
+  late int keysBeforeCompaction;
```

Comment:
is keyBeforeCompaction and sizeBeforeCompaction same or different? Also please add dart docs for the method.

Reply:
sizeBeforeCompaction in size and keysBeforeCompaction is keys ( getSize() and entriesCount() defined in atLogType were used )